### ISSUE TITLE
Review verb construction of menu options

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#864]
@meshanw pointed out that Paying sounds weird in the menu, but it matches the construction of Reporting.  Since we didn't have a content strategist, we didn't do a thorough look of verb construction in the menu options, so we should do that now.

**Comment by mcharg** [comment]
@daniel-coleman-1 Something to consider as you're becoming familiar with the menus while doing the sitemap.

**Comment by meshanw** [comment]
Reporting could be 'Report' or 'How to Report' or 'How to Report to ONRR' or 'Report Royalties.' Then in the drop down Reporting Home could be ' {Insert new header here} Home."

Then Paying can be 'Pay.' "Paying Home" in the drop down can be "Payment Home" or "Payment Overview" 

The other headers and dropdown items look fine to me.

**Comment by daniel-coleman-1** [comment]
I'm getting the impression that every item in the menu (first and second level) is a noun...except Reporting and Paying (I guess you could argue that they are being used as nouns but nevertheless...).

I like "Report/Pay" in the top, and keep "Payments" the same as it is now.

I was thinking of adding Resources and having:
"Production Reporting Resources"
"Royalty Reporting Resources"
"Solid Minerals Reporting Resources"
"Payment Resources"

but I think that makes the menu a bit too wordy.  We don't want our users to have to read through a big block of text, I think we want as simple as possible.  I think simplifying "Reporting/Paying" to "Report/Pay" makes the fix while keeping things simple and clear, what do you guys think?

**Comment by mcharg** [comment]
@daniel-coleman-1 I think you're looking at the categories in the live site.  You should be looking at the categories in the redesign wireframes.
https://qmcq5k.axshare.com/

**Comment by Maroyafaied** [comment]
@daniel-coleman-1 the same with this issue, it's not assigned to this sprint so it can wait till it's assigned.

**Comment by daniel-coleman-1** [comment]
@mcharg you're right, I was on the live site, thanks for the link!

@Maroyafaied sounds good
